scripts/curate_data_axondeepseg_tem.py

```diff
@@ -63,17 +63,21 @@ def main(root_data, output_data):
         contents_subdir = os.listdir(path_subdir)
         sub_bids_full = sub_id + "_sample-" + sample_id
         for file in contents_subdir:
+            # flag_go = False
```

Comment:
Was this accidentally commented out? As written, this will always copy every file still.

Reply:
yes, it was an accident. fixed in the new push